src/interfaces/wallet.cpp

```diff
@@ -488,6 +488,10 @@ class WalletImpl : public Wallet
     {
         return MakeHandler(m_wallet.NotifyWatchonlyChanged.connect(fn));
     }
+    std::unique_ptr<Handler> handleCanGetAddressesChanged(CanGetAddressesChangedFn fn) override
```

Comment:
`handleCanGetAddressesChanged` is a terrible method name in itself, took me at least 5 minutes to decode it, please add a comment saying it's a handler for changes in whether addresses can be generated or not :-)

Reply:
That's a change from #15225 and out of scope for this pr. sorry.